rectools/models/nn/transformers/base.py

```diff
@@ -40,6 +40,7 @@
 )
```

Comment:
Please add changelog, docstrings to SASRecModel and BERT4RecModel and fix tests.